tests/test_vocabulary_handler.py

```diff
@@ -0,0 +1,275 @@
+from lxml import objectify, etree
+import pytest
+
+from controlled_vocabularies.vocabulary_handler import VocabularyHandler
+from . import factories
+
+pytestmark = pytest.mark.django_db
+
+# Namespaces
+NS = {
+    'rdf': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#',
+    'dc': 'http://purl.org/dc/elements/1.1/',
+    'rdfs': 'http://www.w3.org/2000/01/rdf-schema#',
+    'purl': 'http://purl.org/NET/UNTL/vocabularies/formats/'
```

Comment:
Is this a namespace? Perhaps this is suppose to be
```python
# ...
'dcq': 'http://purl.org/dc/terms'
# ...
```
I gathered this from the dumping one of the etree objects.

```xml
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcq="http://purl.org/dc/terms/" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
  <rdf:Description rdf:about="http://purl.org/NET/UNTL/vocabularies/formats/">
    <dc:title>pxWlxuFLPagN</dc:title>
    <dc:publisher>University of North Texas Libraries</dc:publisher>
    <dc:description>KSbEzwNIzRaA</dc:description>
    <dc:language>English</dc:language>
    <dc:date>2013</dc:date>
  </rdf:Description>
  <rdf:Property rdf:about="http://purl.org/NET/UNTL/vocabularies/formats/#LkcopwXXJaCEQBLNeibTWjKVS">
    <rdfs:label>eVFQiYPnRUEauxMbieYuPxPFdkWkkcwdMDokZBNdtYScUFmBfDkjFBuOMAzDdELpRoESFRinJtHeinFkJvJQQIapSvXPbcdkKUYVGDVtpChnFbtYbBCPqVVk</rdfs:label>
    <dc:description>TwDjtKxKcdbK</dc:description>
    <rdfs:isDefinedBy rdf:resource="http://purl.org/NET/UNTL/vocabularies/formats/"/>
  </rdf:Property>
</rdf:RDF>
```

It looks like we might still need the `purl` url, but we are better served by moving it to a constant if it is not truly a namespace.